manifests/stage-ofed-driver/0050_ofed-driver-ds.yaml

```diff
@@ -66,6 +66,12 @@ spec:
               value: {{ .RuntimeSpec.HTTPSProxy }}
             - name: NO_PROXY
               value: {{ .RuntimeSpec.NoProxy }}
+          {{- if .CrSpec.Env }}
```

Comment:
v1.env you use supports also valueFrom key which can be used to reference values from object or from configmap/secret

to support those as well you can just append to the list each entry in `.CrSpec.Env`